statsmodels/duration/tests/test_phreg.py

```diff
@@ -374,6 +403,24 @@ def plf(params):
                 llf_sm = plf(sm_result.params)
                 assert_equal(np.sign(llf_sm - llf_r), 1)
 
+    # Simulation based test of survival quantiles.
```

Comment:
I don't think this belongs in the PR.  It is already merged to master.  May need to rebase and cherry pick unwanted commits.